### ISSUE TITLE
pypi install - missing submodules and data files

### ISSUE THREAD

**Issue by ava-dylang** [richrd/suplemon#135]
I suspect something wrong with the setup.py, 
Probably the data file packaging is wrong, as well as the sub-module inclusion, 

I recommend looking at https://github.com/masci/pypi-example for reference,
data files may need to use lookups like http://stackoverflow.com/a/5899643/1695680

---

Program installed like:
```
mkdir ~/opt; cd ~/opt
python3 -m venv suplemon
./suplmon/bin/pip install suplemon
```
Version:
```
~/opt/suplemon/bin/suplemon --version
0.1.50
```

Trying to launch the program yeilds:
```
2016-03-17 12:33:27,434 - root - DEBUG - Starting Suplemon...
2016-03-17 12:33:27,434 - suplemon.config - ERROR - Failed to load default config file '/home/me/opt/suplemon/lib/python3.4/site-packages/suplemon/config/defaults.json'!
```

The "config", "linelight", "modules", and "themes" directories in `~/opt/suplemon/lib/python3.4/site-packages/suplemon/` don't exist, and I didn't see them anywhere in the install.

**Comment by richrd** [comment]
Oh no, that's unfortunate. I actually seem to have the same problem. I'll get this fixed asap.

**Comment by twolfson** [comment]
@richrd I recently had a similar issue in `flake8-quotes`. Here's how I reproduced the error there:

```bash
# Create temporary flake8-quotes install
mkvirtualenv flake8-quotes
git checkout master

# Build `sdist` file as we do in `release.sh`
python setup.py sdist

# Navigate to parent directory to assure no issues
cd ..

# Install distribution file
pip install flake8_quotes/dist/flake8-quotes-0.2.3.tar.gz

# Verify error is reproduced
flake8 flake8_quotes/

# Try/rinse/repeat with different edits and recompiles
```

**Comment by richrd** [comment]
I actually had the same problem even without virtualenv. It was apparently due to `sdist` not including the package files specified in `setup.py` (in `package_data`). I seem to have it solved by adding the files to `MANIFEST.in`. I've done this in #134 and hope to release soon.

**Comment by twolfson** [comment]
The `virtualenv` is mostly there to prevent contaminating the global install. Glad to hear the issue has been resolved =)

**Comment by richrd** [comment]
Fixed in the new release v0.1.51, hope it works for you now.
